src/main/scala/com/atomist/rug/function/github/issue/ListUserIssuesFunction.scala

```diff
@@ -27,25 +24,27 @@ class ListUserIssuesFunction extends AnnotatedRugFunction
 
     logger.info(s"Invoking listUserIssues with days '$days' and token '${safeToken(token)}'")
 
-    Try {
+    try {
```

Comment:
Hmm, from memory, the Failure(t) was still letting through GitHub error conditions, and so I used try/catch and hence my comment in the catch block. I'll do some more testing. I'd like to be consistent